src/manager/docker.service.ts

```diff
@@ -65,15 +71,30 @@ export class DockerService implements IAgentManager {
                 '--wallet-local-did', // TODO this could be an arg on the config
             ],
         };
+
+        // allow for exposing admin ports when set as this might be needed during development.
+        // to make the ports available to localhost, they need to be set to values other than
+        // the defaults
+        if (config.adminPort !== process.env.AGENT_ADMIN_PORT && Constants.LOCAL === process.env.NODE_ENV) {
```

Comment:
When the condition is true and we update ExposedPorts, the call `this.dockerode.createContainer(containerOptions);` (below) hangs, no agent is created and no exceptions .  We will never see `Logger.warn(`starting container`);`